bundles/paikkatietoikkuna/coordinateconversion/components/table.js

```diff
@@ -0,0 +1,191 @@
+Oskari.clazz.define('Oskari.coordinateconversion.component.table', function( instance, loc ) {
+    this.instance = instance;
+    this.loc = loc;
+    this.numrows = 1;
+    this.container = null;
+    this.template = {
+            tableWrapper: jQuery('<div class="table"></div>'),
+            rowcounter: _.template('<div class="rowcount"><span id="num"></span> <%= rows %> </div>'),
+            header: _.template('<div class="oskari-table-header">'+
+                                        '<table id="oskari-tbl-header" cellpadding="0" cellspacing="0" border="0">'+
+                                            '<thead>'+
+                                                '<tr>' +
+                                                    '<th id="nort"><%= north %></th>'+
+                                                    '<th id="east"><%= east %></th>'+
+                                                    '<th id="ellipse_height" ><%= ellipse_height %></th>'+
+                                                '</tr>'+
+                                             '</thead>'+
+                                        ' </table>'+
+                                     '</div>'),
+            row: _.template('<tr>' +
+                                    '<td class="cell lon" headers="north" style=" border: 1px solid black ;"> <%= coords.lon %> </td>'+
+                                    '<td class="cell lat" headers="east" style=" border: 1px solid black ;"> <%= coords.lat %> </td>'+
+                                    '<td class="cell height" headers="ellipse_height" style=" border: 1px solid black;"></td>'+
+                                    '<td class="cell control"> <div class="removerow"></div></td>'+
+                                '</tr> '),
+            table:_.template('<div class="coordinatefield-table" style="display:inline-block;">' +
+                                        '<h5> <%= input %> </h5>' +
+                                        '<div class="oskari-table-content">'+
+                                         '<div style="width:100%; overflow:auto;">'+
+                                            '<table class="hoverable" id="oskari-coordinate-table" style="border: 1px solid black;" cellpadding="0" cellspacing="0" border="0">'+
```

Comment:
Having two of these tables in the UI results in multiple DOM elements having the same id.